README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# HOMEWORK ONE   
+## Who is Paulino?
+![](https://github.com/pdiazm/hw01/blob/master/PD_Headshot.jpg)
```

Comment:
It is great that you added the photo to your repo! It would have made it more accessible and easy to add it to the code, which was something I had struggled with.